crates/ruma-common/src/push/predefined.rs

```diff
@@ -325,6 +327,28 @@ impl ConditionalPushRule {
             }],
         }
     }
+
+    /// Matches a poll response event sent in any room.
+    ///
+    /// This rule should be kept in sync with `.m.rule.message` by the server.
```

Comment:
The msc says that syncing of the poll rules is up to the client.

Reply:
Indeed, I forgot to remove that